src/derivation.c

```diff
@@ -0,0 +1,107 @@
+/* Counterexample derivation trees
+ 
+ Copyright (C) 2019-2020 Free Software Foundation, Inc.
```

Comment:
We should stick to "2020" for all the files you contributed.  We can address that later.